src/plugins/gopts/README.md

```diff
@@ -0,0 +1,34 @@
+- infos = Information about the gopts plugin is in keys below
+- infos/author = Klemens Böswirth <k.boeswirth+git@gmail.com>
+- infos/licence = BSD
+- infos/needs =
+- infos/provides =
+- infos/recommends =
+- infos/placements = procgetstorage
+- infos/status = recommended productive maintained nodep libc experimental
+- infos/metadata =
+- infos/description = Parses command-line options using elektra-opts
+
+## Introduction
+
+This plugin is very simple in what it does. It uses system specific methods of accessing `argc` and `argv` outside of `main` and then calls
+`elektraGetOpts` to do options processing.
+
+## Usage
+
+The preferred way of using this plugin is via `kdbEnsure`:
```

Comment:
Isn't global mounting possible, too?

Reply:
Technically yes, but the `kdb` tool doesn't call `kdbEnsure` yet (to make sure `gopts` is unmounted, when accessing third-party configuration), so `kdb get` etc may break in certain cases.

`gopts` is unmounted by default for now. My plan was to make it mounted by default in future and just unmount it in `kdb`. That is however quite tedious work, because each command uses its own `KDB` instance, not created through some shared means.

Reply:
What about subclassing KDB (where `gopts` is ensured to be not there) and replace all KDB with the subclass? (Should be a simply search&replace.)

Reply:
I think it would be better to create `KDBCommand`, an abstract subclass of `Command` that provides a `KDB` instance to its subclasses. I think this is a nicer solution, because later on KDB should use `gopts` for its own config and only the commands accessing the rest of the KDB should use the version without `gopts`.

I will modify `kdb` in a follow-up PR and then we might mount `gopts` by default.